buildSrc/src/main/kotlin/Versions.kt

```diff
@@ -16,6 +16,7 @@
 
 object Versions {
     const val KOTLIN_VERSON = "1.4.10"
+	const val KAPT_VERSION = "1.4.10"
```

Comment:
There is a more recent version but when I moved it and Kotlin version out of sync - it did not end well.